include/dfu/suit_dfu.h

```diff
@@ -48,9 +47,11 @@ int suit_dfu_initialize(void);
 
 /**
  * @brief Purge all the SUIT DFU partitions and perform cleanup.
- *        Call this if the update needs to be interrupted after suit_dfu_initialize is called.
- *        This will invalidate any written data. To start the update again, SUIT DFU must
- *        be reinitialized with suit_dfu_envelope_size_update.
+ *
+ * @details
+ * Call this if the update needs to be interrupted after @ref suit_dfu_candidate_envelope_stored
+ * is called. This will invalidate any written data. To start the update again, SUIT DFU must
```

Comment:
"any written data" is ambiguous here. Does it invalidate any data written by `suite_dfu_candidate_envelope_stored` or does it purge the whole `dfu_parition` space, including the downloaded candidate?

Reply:
Do I understand correctly that the envelope must be restored in the dfu_partition after calling this function (probably re-downloaded)? If so, maybe it would be good to state it alongside the need to call `suit_dfu_candidate_envelope_stored`?

Reply:
Description extended.